src/runtime/virtcontainers/types/dan.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+type DanConfig struct {
+	Netns   *string     `json:"netns"`
```

Comment:
Looks like the Netns is never used?

Reply:
this is mirrored from kata-runtime-rs's DanConfig definition, it's not used -- ovn-k8s CNI sets it to host netns.

Reply:
Got it. But I still think you should implement the netns check and leave some comments here, even though it appears to be unnecessary at the moment. Otherwise, others will be confused about the useless field.

```go
if (danConfig.Netns != nil) {
    // Go into the netns but it hasn't been implemented yet.
    // panic here
}
```

Reply:
For VFIO network device, it doesn't present as a network interface on the host -- that's, no netns operation required on host-side. How about we revisit it when other kind of network interfaces involved?

Reply:
Fair enough. Leaving a panic is not a good practice.